_episodes/04-higgs-search.md

```diff
@@ -311,7 +311,7 @@ Moreover, we can compare, by printing, the number of initial  and final events a
 
 ```python
 for s in samples:
-    print(s, "      Initial events: ", len(branches[s]["m4l"]))
+    print(f"{s}       Initial events:  {len(branches[s]['m4l'])}")
```

Comment:
Let's make this slightly fancier and add `:,` for all long ints that we print, e.g., `{len(branches[s]['m4l']):,}`. It's a great win for readability!

Reply:
Same with the after selection below and the stack_...

Reply:
@Vaibhav-Chopra-GT